app/controllers/concerns/api/csrf_token_concern.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  module CsrfTokenConcern
+    def include_csrf_token_header
```

Comment:
WDYT of calling this `add_csrf_token_header_to_response`?
```suggestion
    def add_csrf_token_header_to_response
```

Reply:
Works for me!

Reply:
> WDYT of calling this `add_csrf_token_header_to_response`?

Updated in 722233293